src/xlsb/cells_reader.rs

```diff
@@ -11,7 +11,7 @@ use super::{cell_format, parse_formula, wide_str, RecordIter};
 pub struct XlsbCellsReader<'a> {
     iter: RecordIter<'a>,
     formats: &'a [CellFormat],
-    strings: &'a [String],
+    strings: &'a [RichText],
```

Comment:
I think it is wrong to force all strings to be `RichText`. There are already some situation where memory is an issue.

Reply:
Do you have an idea how to handle this? We need to store either normal strings or rich text, both can be the case at any shared string. So naturally you would make an enum:
```rust
enum SharedString {
    String(String),
    RichText(RichText),
}
```
But then the enum type is at least as big as RichText, if not more. So using `&[SharedString]` does not solve the memory issue.

So how are we able to store rich text while keeping the memory layout of string in the usual case? We could make the rich text varian `RichText(Box<RichText>)`, but that has different overhead, that I am not sure if it is better than additional memory use?